BEIMA.Client/services/ApiClient.js

```diff
@@ -0,0 +1,32 @@
+//pull in the vars file to get the API URL
+require('dotenv').config({ path: '../.env' })
+
+//axios package includes it's own try catch and error output functions 
+
+const axios = require('axios');
+
+const API_URL = process.env.REACT_APP_API_URL;
+
+/// <summary>
```

Comment:
I know this is our header formatting for our C# code, but do we want to use the same formatting here?

Reply:
All the examples I see after looking to see if there's a standard appear to be using the JSDoc format.
https://en.wikipedia.org/wiki/JSDoc

I'll swap it for that since it seems to be nearly ubiquitous